protected/humhub/components/behaviors/AccessControl.php

```diff
@@ -146,7 +146,10 @@ public function beforeAction($action)
         $this->controllerAccess = $this->getControllerAccess($this->rules);
 
         if (!$this->controllerAccess->run()) {
-            if ($this->controllerAccess->code == 401) {
+            if (isset($this->controllerAccess->codeCallback) &&
```

Comment:
Could you add some further docs for usage and use-cases of `codeCallback`

Reply:
I added a doc right in the place, commit https://github.com/humhub/humhub/pull/4602/commits/003cc56f842f7fb0471ee1f812324476f89af444,
but doc is also in declaration https://github.com/humhub/humhub/pull/4602/files#diff-230df3417b1e6af913d2567248216004090ea2735ad49c67502cc3a2262ad5acR205.